library/ssl_cli.c

```diff
@@ -3316,7 +3316,7 @@ static int ssl_parse_new_session_ticket( mbedtls_ssl_context *ssl )
 
     msg = ssl->in_msg + mbedtls_ssl_hs_hdr_len( ssl );
 
-    lifetime = ( msg[0] << 24 ) | ( msg[1] << 16 ) |
+    lifetime = ( ((uint32_t) msg[0]) << 24 ) | ( msg[1] << 16 ) |
                ( msg[2] <<  8 ) | ( msg[3]       );
```

Comment:
There was some quirky spacing here to give alignment of brackets. (Bit unnecessary but looked pretty). Could you remove the excessive spacing? It looks odd now the alignment has been broken.

Reply:
Doing it and squashing the commit together